styles.css

```diff
@@ -0,0 +1,1234 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap');
+
+* {
+  font-family: poppins, Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  box-sizing: border-box;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-content: center;
+  height: 8vh;
+  width: 70vw;
+  margin-left: 0.2rem;
+  padding-top: 8px;
+}
+
+.header .menu-icon {
+  position: relative;
+  top: 10%;
+  margin-right: 30%;
+  width: 0.7rem;
+  height: 0.7rem;
+  left: 5%;
+}
+
+.header .website-logo {
+  width: 2.4rem;
+  height: 1rem;
+  position: relative;
+  top: -0.2rem;
+}
```

Comment:
Great design @marvin-nyalik 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the given design. You may remove the logo from your design so that your design can look like the design given.
- Your design
![Your-home-mobile](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/05b7cebc-aae1-492c-954a-578f98089597)
- Given design
![Given-home-mobile](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/2fba98f9-fe3b-42e4-aacc-b7b984202be7)